### PR TITLE
Update: Microsoft.WingetCreate version 1.1.2.0

### PR DESCRIPTION
**Pull request by russellbanks**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

This adds the portable wingetcreate exe

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/69431)

### CODE REVIEW

**Review by ItzLevvie — DISMISSED** [review]

File: manifests/m/Microsoft/WingetCreate/1.1.2.0/Microsoft.WingetCreate.locale.en-US.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
 PackageIdentifier: Microsoft.WingetCreate
 PackageVersion: 1.1.2.0
 PackageLocale: en-US
 Publisher: Microsoft Corporation
-PublisherUrl: https://www.microsoft.com/en-US/
+PublisherUrl: https://www.microsoft.com
 PublisherSupportUrl: https://github.com/microsoft/winget-create/issues
-PrivacyUrl: https://privacy.microsoft.com/en-US/privacystatement
+PrivacyUrl: https://github.com/microsoft/winget-create/blob/main/PRIVACY.md
```
Comment (Trenly) [inline manifests/m/Microsoft/WingetCreate/1.1.2.0/Microsoft.WingetCreate.locale.en-US.yaml:10]:
```suggestion
PrivacyUrl: https://privacy.microsoft.com/en-US/privacystatement
```
The original URL is correct, as the GitHub one just tells users to go to the microsoft.com one

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by Trenly** [comment]
As I've been thinking. . . @ryfu-msft / @denelon - Given that this package is managed by the winget team, do you want the portable exe added here?

**Comment by denelon** [comment]
I've been thinking about this one since we have the MSIX version and the portable versions with and without dependencies to support CI/CD flows. We don't have anything like `winget install <package> --installer-type [MSIX, MSI, .exe, portable, zip]`. This is likely to set a precedent so we should be intentional.

**Comment by denelon** [comment]
We should look at:
* https://github.com/microsoft/winget-cli/issues/1166

It might make sense to have Microsoft.WingetCreate.Portable and break that down into the versions with and without dependencies. The manifest for the .exe without dependencies should declare them so if they are on the system they don't get downloaded.

**Comment by russellbanks** [comment]
> It might make sense to have Microsoft.WingetCreate.Portable and break that down into the versions with and without dependencies. The manifest for the .exe without dependencies should declare them so if they are on the system they don't get downloaded.

Haven't the self-contained portable .exe's been removed since v1.1.2.0:
> Note: Self-contained exes have been removed since WS2022 now supports AppExecutionAliases. Check out the [README](https://github.com/microsoft/winget-create#using-the-msixbundle) for instructions on how to use the msixbundle in your CI/CD pipeline.

**Comment by denelon** [comment]
I think I'm OK having both the MSIX and the portable .exe in the same manifest. It's going to take feature work to allow someone to install the portable though. It might be best in the short term to have a Microsoft.WingetCreate.Portable manifest for now so people can use winget to install the portable version.

**Comment by denelon** [comment]
@ryfu-msft let me know if you have a different opinion 😊

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**